Tests/CMakeLists.txt

```diff
@@ -210,10 +219,10 @@ endfunction(add_test_spray)
 #=============================================================================
 
 if(NOT PELE_ENABLE_EB)
-  add_test_r(flamesheet-drm19-${PELE_DIM}d FlameSheet)
+  add_test_re(flamesheet-drm19-${PELE_DIM}d FlameSheet)
```

Comment:
These shouldn't be excluded from the CI. If they're taking too long we should reduce the size instead.

Reply:
Ok, maybe I will do what we in PeleC and only run like 1 test under `Debug`.